docs/installation/setup.rst

```diff
@@ -37,18 +37,18 @@ Provisioning
 
 SSH
 ^^^
-Each machine should be accessible through SSH from your host. As part of the
+Each machine must be accessible through SSH from the host. As part of the
 :doc:`./bootstrap`, a new SSH identity for the :term:`Bootstrap node` will be
 generated and shared to other nodes in the cluster. It is also possible to do
 it beforehand.
 
 Network
 ^^^^^^^
-Each machine needs to be a member of both the control-plane and workload-plane
-networks, as described in :ref:`quickstart-intro-networks`. However, these
+Each machine must be a member of both the control-plane and workload-plane
+networks, as described in :ref:`installation-intro-networks`. However, these
 networks can overlap, and nodes need not have distinct IPs for each plane.
 
-In order to reach the cluster-provided UIs from your host, the host needs to be
+For the host to reach the cluster-provided UIs, it must be
 able to connect to workload-plane IPs of the machines.
```

Comment:
Uh, this was not updated recently... Could you replace `workload-plane` with `control-plane` here please?